src/VisualStudio/Core/Impl/CodeModel/CodeModelEventQueue.cs

```diff
@@ -35,6 +35,7 @@ private void EnqueueEvent(CodeModelEvent @event)
             {
                 var priorEvent = _eventQueue.Peek();
                 if (priorEvent.Node == @event.Node &&
+                    priorEvent.ParentNode == @event.ParentNode &&
```

Comment:
We have cases. In the below code
```
<Attr1, Attr2>
int v1,v2;
```

If we remove the Attribute `Attr2`
```
<Attr1>
int v1,v2;
```

Then we need to issue 2 remove events, where the node(`Attr2`) is the same but the parent nodes are `v1` and `v2` resp. The testcases I have written asserts this change.